### PR TITLE
docs: Add xpath and grep to npm packages documentation

### PR DESCRIPTION
**Pull request by jordanpowell88**
Adds `@cypress/xpath` and `@cypress/grep` to the table of `npm` packages

- Closes  https://github.com/cypress-io/cypress/pull/23887#issuecomment-1255106926

### Additional details
Updates Documentation for the `@cypress/xpath` & `@cypress/grep` packages

### PR Tasks

- [ ] Have tests been added/updated?
- [ ] Has the original issue (or this PR, if no issue exists) been tagged with a release in ZenHub? (user-facing changes only)
- [ ] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [ ] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -169,6 +169,8 @@ Here is a list of the npm packages in this repository:
  | [webpack-dev-server](./npm/webpack-dev-server)     | `@cypress/webpack-dev-server`    | Webpack powered dev server for Component Testing.                |
  | [vue](./npm/vue)                                       | `@cypress/vue`                     | Cypress component testing for Vue 3.               |
  | [vue2](./npm/vue2)                                       | `@cypress/vue2`                     | Cypress component testing for Vue 2.               |
+ | [xpath](./npm/xpath)                                     | `@cypress/xpath`                    | Adds XPath command to Cypress.io test runner        |
+ | [grep](./npm/grep)                                       | `@cypress/grep`                     | Filter tests using substring                        |
```
Comment (emilyrohrbough) [inline CONTRIBUTING.md]: nit: abc order

**Review by emilyrohrbough — APPROVED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (admah) [reply CONTRIBUTING.md]: @jordanpowell88 can you also add the Schematic to this list? I noticed it's missing too.

Comment (jordanpowell88) [reply CONTRIBUTING.md]: @admah Done in fcfa6c31d7de842a0725bf4bc521c8956464bcbd

**Review by emilyrohrbough — APPROVED** [review]

**Review by jaffrepaul — APPROVED** [review]
LGTM 🎉

**Pull request merged**